package.json

```diff
@@ -8,10 +8,10 @@
   },
   "dependencies": {
     "acorn": "0.4.0",
+    "addons-linter": "0.6.0",
```

Comment:
I did that, but this is just me putting it in alphabetical order because I'm OCD. I can take it out if you like though.